source/sync/data-access-patterns/flexible-sync-permissions-guide.txt

```diff
@@ -305,7 +305,7 @@ of authors in my custom user data."
 .. note:: Size Limitations
 
    In this example, we are creating an array in the Custom User Data. The 
-   size of this array is not limited by {+realm+}, but because the data is 
+   size of this array is not limited by realm, but because the data is 
```

Comment:
lowercase 'realm' is def incorrect here. i think 'Device Sync' or 'App Services' would be correct here.

Reply:
Replaced with 'App Services'